src/lib.rs

```diff
@@ -568,24 +582,37 @@ impl VectorSelector {
     }
 }
 
-extension_sql!(
-    r#"
-CREATE AGGREGATE vector_selector(
-    start_time timestamptz,
-    end_time timestamptz,
-    bucket_width bigint,
-    lookback bigint,
-    sample_time timestamptz,
-    sample_value DOUBLE PRECISION)
-(
-    sfunc = vector_selector_transition,
-    stype = internal,
-    finalfunc = vector_selector_final,
-    combinefunc = vector_selector_combine,
-    serialfunc = vector_selector_serialize,
-    deserialfunc = vector_selector_deserialize,
-    parallel = safe
-);
-"#,
-    name = "create_aggregate_vector_selector"
-);
+// extension_sql!(
+//     r#"
+// CREATE AGGREGATE vector_selector(
+//     start_time timestamptz,
+//     end_time timestamptz,
+//     bucket_width bigint,
+//     lookback bigint,
+//     sample_time timestamptz,
+//     sample_value DOUBLE PRECISION)
+// (
+//     sfunc = vector_selector_transition,
+//     stype = internal,
+//     finalfunc = vector_selector_final,
+//     combinefunc = vector_selector_combine,
+//     serialfunc = vector_selector_serialize,
+//     deserialfunc = vector_selector_deserialize,
+//     parallel = safe
+// );
+// "#,
+//     name = "create_aggregate_vector_selector"
+// );
+
+#[cfg(test)]
+#[pg_schema]
+pub mod pg_test {
```

Comment:
this looks like it does nothing?

Reply:
It does do something, and what it does is interesting.

Basically when you write `pgx` tests, the test functions are executed _inside_ the database. pgx compiles the test function down, creates SQL schema for it, and installs the functions into the db. The functions are all "installed" into the `pg_test` DB schema. The `#[pg_schema]` here ensures that `pub mod pg_test` gets interpreted as "create a new DB schema named `pg_test`", for the test functions to get installed into.